docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -0,0 +1,1249 @@
+        },</programlisting>
+
+      <para>Note the <emphasis>additionalProperties</emphasis> value. In the
+      above example it allows for additional properties. If that
+      additionalProperties value is present and set to false(example below),
+      then no other properties are allowed and the property list is
+      complete.<programlisting>    "additionalProperties": false</programlisting>In
+      working with the HPCC Systems values.yaml, the values file must validate
+      against this schema. If there is a value that is not allowed as defined
+      in the schema file it will not start and instead generate an
+      ERROR.</para>
+    </sect2>
+
+    <sect2 id="TheValuesYamlFileStruct">
+      <title>The values.yaml file structure</title>
+
+      <para>The values.yaml file is a yaml file. Yaml is a data serialization
+      language often used as a format for configuration files. A key construct
+      that makes up the bulk of a yaml file is the key-value pair, sometimes
+      referred to as a hash or a dictionary. The key-value pair construct
+      consists of a key that points to some value(s). The values could be
+      strings, numbers, booleans, integers, arrays, or dictionaries, and
+      lists.</para>
+
+      <para>In yaml files the indentation is used to represent document
+      structure and nesting. Leading spaces are significant and tabs are not
+      allowed.</para>
+
+      <sect3 id="YAML_Dictionary">
+        <title>Dictionary</title>
+
+        <para>Dictionaries are collections of key value mappings. All keys are
+        case-sensitive and as we mentioned earlier the indentation is also
+        crucial. These keys must be followed by a colon (:) and a space.
+        Dictionaries can also be nested.</para>
+
+        <para>Dictionary is a key: value, followed by another key: value:, for
+        example:</para>
+
+        <programlisting>    logging: 
+       detail: 80 
+</programlisting>
+
+        <para>This (logging) is an example of a dictionary.</para>
+
+        <para>Dictionaries in passed in values files, such as the
+        <emphasis>myoverrides.yaml</emphasis> file in the example below, will
+        merge values.</para>
+
+        <para><programlisting>helm install myhpcc hpcc/hpcc -f myoverrides.yaml</programlisting>In
+        the case of a merged values, any values not overridden remain. You
+        can't necessarily replace or simply delete a dictionary, If you wanted
+        to wipe it out you can set an existing dictionary to null which will
+        then wipe it.</para>
+      </sect3>
+
+      <sect3 id="YAML_Lists">
+        <title>Lists</title>
+
+        <para>Lists are groups of elements beginning at the same indentation
+        level starting with a - (a dash and a space). Every element of the
+        list is indented and starts with a dash and a space. Lists can also be
+        nested.</para>
+
+        <para>An example of a nested list looks like:</para>
```

Comment:
An example of a list of dictionaries, with placement.tolerations as a nested list.